roles/matrix_mult/vars/main.yml

```diff
@@ -0,0 +1,5 @@
+matrix_mult_cm_entrypoint: roles/matrix_mult/files/matrix_mult_cm_entrypoint.yml
+matrix_mult_pod: roles/matrix_mult/files/matrix_mult_pod.yml
+
+# time argument for matrix mult, in seconds
+matrix_mult_time: 300
```

Comment:
```
# time argument for matrix mult, in seconds
matrix_mult_time: 300
```
this can go in `roles/<role>/defaults/main/config.yml`

and 
```
matrix_mult_cm_entrypoint: roles/matrix_mult/files/matrix_mult_cm_entrypoint.yml
matrix_mult_pod: roles/matrix_mult/files/matrix_mult_pod.yml
```
in `roles/<role>/vars/main/resources.yml`